flake.nix

```diff
@@ -132,28 +142,46 @@
           The `overlays` flake output defines ["overlays"](https://nixos.org/manual/nixpkgs/stable/#chap-overlays) that can be plugged into Nixpkgs.
           Overlays add additional packages or modify or replace existing packages.
         '';
-        inventory = output: mkChildren (builtins.mapAttrs (overlayName: overlay:
-          {
-            what = "Nixpkgs overlay";
-            evalChecks.isOverlay =
-              # FIXME: should try to apply the overlay to an actual
-              # Nixpkgs.  But we don't have access to a nixpkgs
-              # flake here. Maybe this schema should be moved to the
-              # nixpkgs flake, where it does have access.
-              builtins.isAttrs (overlay {} {});
-          }) output);
+        inventory = output: mkChildren (builtins.mapAttrs
+          (overlayName: overlay:
+            {
+              what = "Nixpkgs overlay";
+              evalChecks.isOverlay =
+                # FIXME: should try to apply the overlay to an actual
+                # Nixpkgs.  But we don't have access to a nixpkgs
+                # flake here. Maybe this schema should be moved to the
+                # nixpkgs flake, where it does have access.
+                builtins.isAttrs (overlay { } { });
+            })
+          output);
       };
 
       nixosConfigurationsSchema = {
         version = 1;
         doc = ''
           The `nixosConfigurations` flake output defines [NixOS system configurations](https://nixos.org/manual/nixos/stable/#ch-configuration).
         '';
-        inventory = output: mkChildren (builtins.mapAttrs (configName: machine:
-          {
-            what = "NixOS configuration";
-            derivation = machine.config.system.build.toplevel;
-          }) output);
+        inventory = output: mkChildren (builtins.mapAttrs
+          (configName: machine:
+            {
+              what = "NixOS configuration";
+              derivation = machine.config.system.build.toplevel;
+            })
+          output);
+      };
+
+      homeConfigurationsSchema = drvSet {
```

Comment:
Aren't `homeConfigurations` and `darwinConfigurations` the result of a call to `evalModules`, i.e. an attrset that contains an attribute like `config.system.build.toplevel`? If so the inventory function should be similar to `nixosConfigurationsSchema`.

Reply:
Indeed this doesn't work, the derivation for Home Manager should point to `activationPackage`. Example: https://github.com/gvolpe/flake-schemas/blob/7d762079449d0ca63e92b0128c885021168c0c79/flake.nix#L242